src/corelibs/U2View/src/ov_sequence/AnnotatedDNAView.cpp

```diff
@@ -469,7 +469,7 @@ bool AnnotatedDNAView::onObjectRemoved(GObject* o) {
             emit si_sequenceRemoved(seqCtx);
             seqContexts.removeOne(seqCtx);
             removeAutoAnnotations(seqCtx);
-            delete seqCtx;
+            seqCtx->deleteLater();
```

Comment:
This should not be needed anymore, right? 
The only fix needed is that QTimer line that moves the deletion of the the nested event processing.

Reply:
Both options work, I like `deleteLater`, but I removed it.